### PR TITLE
Signup form

### PR DESCRIPTION
**Pull request by epicallan**

### CODE REVIEW

File: src/views/components/Signup_form/index.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import { Button, Checkbox, Form } from "semantic-ui-react";
+
+export interface SignUpProps {
+  autoFocus?: boolean;
+  loading?: boolean;
+  onChange: (term: string) => void;
+}
+
+interface SignUpState {
+  firstName: " ";
```
Comment (epicallan) [inline src/views/components/Signup_form/index.tsx]: we are only asking for userName. No need for firstName and LastName. 
Also, specify actual types, not string literals. i.e you should have `email: string` not `email: " "`

File: src/views/components/Signup_form/index.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  lastName: " ";
+  email: " ";
+  password: " ";
+}
+
+class SignUp extends React.Component<SignUpProps, SignUpState> {
+  public render() {
+    return (
+      <div>
+        <h1>Sign Up</h1>
```
Comment (epicallan) [inline src/views/components/Signup_form/index.tsx]: use react semantic-ui `Header` component with `as` as h3 https://react.semantic-ui.com/elements/header/#types-page

File: src/views/components/Signup_form/index.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+              type="text"
+              id="email_1"
+              name="email1"
+            />
+          </Form.Field>
+          <Form.Field>
+            <label>Password</label>
+            <input
+              placeholder=" New Password"
+              type="text"
```
Comment (epicallan) [inline src/views/components/Signup_form/index.tsx]: the type should be password. Also had attribute `required={true}`

File: src/views/components/Signup_form/index.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+              id="email"
+              name="email"
+            />
+          </Form.Field>
+          <Form.Field>
+            <label>Re-enter Email</label>
+            <input
+              placeholder="Your email"
+              type="text"
```
Comment (epicallan) [inline src/views/components/Signup_form/index.tsx]: should be type `email` also add `required` attribute set to true

File: src/views/components/Signup_form/index.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+            <label>Password</label>
+            <input
+              placeholder=" New Password"
+              type="text"
+              id="pass_word"
+              name="password"
+            />
+          </Form.Field>
+          <Form.Field>
+            <Checkbox label="I agree to the Terms and Conditions" />
```
Comment (epicallan) [inline src/views/components/Signup_form/index.tsx]: Not required. Please, remove

File: src/views/components/Signup_form/index.tsx [hunk 6]
```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import { Button, Checkbox, Form } from "semantic-ui-react";
+
+export interface SignUpProps {
+  autoFocus?: boolean;
+  loading?: boolean;
+  onChange: (term: string) => void;
```
Comment (epicallan) [inline src/views/components/Signup_form/index.tsx]: this should be onSubmit and should take in a signup data object

File: src/views/components/Signup_form/index.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import { action } from "@storybook/addon-actions";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import SignUp from "./index";
+
+storiesOf("components/signUp", module).add("default", () => (
+  <SignUp autoFocus={true} onChange={action("signUp")} />
+));
```
Comment (epicallan) [inline src/views/components/Signup_form/index.stories.tsx]: Missing empty end of file line see https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

File: src/views/components/Signup_form/index.tsx (hunk 3 above)
Comment (brian-seruwuge) [reply src/views/components/Signup_form/index.tsx]: Should'nt the form have some sort of additional password for validation to check if there is a match between what the user has typed the first time and what he re-enters

Comment (epicallan) [reply src/views/components/Signup_form/index.tsx]: That's good to have you can add it.

File: src/views/components/Signup_form/index.tsx (hunk 5 above)
Comment (brian-seruwuge) [reply src/views/components/Signup_form/index.tsx]: u mean line 69?

Comment (epicallan) [reply src/views/components/Signup_form/index.tsx]: yes

**Pull request closed without merging**